code/datums/mind.dm

```diff
@@ -319,8 +325,11 @@
 		text = "<i><b>[text]</b></i>: "
 		if (src in ticker.mode.cult)
 			text += "loyal|<a href='?src=\ref[src];cult=clear'>employee</a>|<b>CULTIST</b>"
-			text += "<br><a href='?src=\ref[src];cult=equip'>Equip</a>"
-
+			text += "<br>Give <a href='?src=\ref[src];cult=tome'>tome</a>|<a href='?src=\ref[src];cult=amulet'>amulet</a>."
+/*
+			if (objectives.len==0)
+				text += "<br>Objectives are empty! Set to sacrifice and <a href='?src=\ref[src];cult=escape'>escape</a> or <a href='?src=\ref[src];cult=summon'>summon</a>."

```

Comment:
I set its probability to 0, or cut the line that allows it to be added as an objective, I don't recall. Either way I left the rest of the escape code in, in case someone thinks it should be re-added.